packages/admin-ui/src/filters/filter/filter.state.tsx

```diff
@@ -62,7 +68,7 @@ export function useFilterState<T extends FilterItem>(
     if (isMenuClosed && combobox.value) {
       // resets combobox
       combobox.setValue('')
-      combobox.setSelectedItem(appliedItem)
+      combobox.setSelectedItem(appliedItem || undefined)
```

Comment:
Why do you pass undefined here? 🤔

Reply:
That's how you unselect an item. Here when the menu is closed we'll select the last applied item, if there is none we will deselect anything that was selected. This is to erase anything that was selected, but not applied, when the menu closed